util/eventbus/assets/main.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <script src="bus/htmx.min.js"></script>
+        <script src="bus/htmx-websocket.min.js"></script>
+        <link rel="stylesheet" href="bus/style.css">
```

Comment:
I suppose we probably won't ever view this on a browser with a different default, but maybe also `<meta charset="utf-8" />` or similar. It'll be a no-op in Chrome and Firefox anyway, though, so maybe not.

Reply:
I believe this is no longer necessary, as long as the page begins with exactly `<!DOCTYPE html>`. That tells the browser this is an HTML5 document, and for html5 UTF-8 is the default encoding. All current browsers understand the HTML5 doctype, so we should be set until the day we need to debug using an ancient IE6 or somesuch :joy: